app/editor/src/components/form/datepicker/SelectDate.tsx

```diff
@@ -53,7 +53,11 @@ export const SelectDate: React.FC<IDatePickerProps> = ({
 
   return (
     <styled.SelectDate className="frm-in" variant={variant} required={required} width={width}>
-      {label && <label htmlFor={id ?? `txt-${name}`}>{label}</label>}
+      {label && (
+        <label className={required ? 'required' : ''} htmlFor={id ?? `txt-${name}`}>
```

Comment:
You can do this without the extra className.  I believe the parent object will have the attribute `required`, it'll also be available in the styled component.

Reply:
Will take a look at that

Reply:
Looks like required attribute does not get added to div elements